cram/cram_io.c

```diff
@@ -4575,3 +4575,27 @@ int cram_set_voption(cram_fd *fd, enum hts_fmt_option opt, va_list args) {
 
     return 0;
 }
+
+#define TEMPLATE_LEN 38
+int cram_check_EOF(cram_fd *fd)
+{
+    uint8_t buf[TEMPLATE_LEN];
```

Comment:
`uint8_t` is only used in this code for interfacing with the lzma library.  Just use `char`, or `unsigned char` if it makes you feel better.